tutorials/animation/2d_skeletons.rst

```diff
@@ -24,7 +24,7 @@ The following tutorial will, then, explain 2D skeletal deformations.
 Setup
 -----
 
-.. seealso:: Before starting, it is advised to go through the :ref:`doc_cutout_animations` tutorial
+.. seealso:: Before starting, it is advised to go through the :ref:`doc_animation_tree` tutorial
```

Comment:
Are you sure about this change? I think @reduz referenced the Cutout animations page on purpose (the Skeletons tutorial was even written before the AnimationTree one existed).

Reply:
Since this tutorial basically redoes the same as Cutout animations using skeletons, I think this is indeed the tutorial that should be referenced (so undo this change).